.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [ '1.8', '11' ]
+    name: Build Parquet with JDK ${{ matrix.java }} and ${{ matrix.codes }}
```

Comment:
`matrix.codes` doesn't seem to point to anything. Just drop it?
(see formatted labels in https://github.com/gszadovszky/parquet-format/runs/1816450871 )